config/insultibot/SampleUtterances.txt

```diff
@@ -4,4 +4,18 @@ INSULT to give me an insult
 BACKHANDED_COMPLIMENT to compliment me
 BACKHANDED_COMPLIMENT for a compliment
 BACKHANDED_COMPLIMENT to give me a compliment
-HELP for a little bit of help
\ No newline at end of file
+HELP for a little bit of help
+HELP how do I use you
+HELP how do I use it
```

Comment:
FWIW, I think that'll have almost no effect in how utterances are triggered.  "I/we/me/my" are essentially garbage words that don't help disambiguate a phrase, so they're not what Alexa is looking for.

I was reading a forum post earlier today where the discussion was that you at a certain point adding more utterances that are almost identical can start being a hindrance rather than a help.